uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/controllers/modal/modal-document.controller.js

```diff
@@ -549,5 +561,16 @@ define([
         $rootScope.$broadcast('ct-spinner-hide');
       });
     }
+
+    /**
+     * Executes watchers for changes
+     */
+    function watchForChanges () {
+      // Watches the changes in cached contacts search array
```

Comment:
unnecessary comment, the code is self explanatory

Reply:
Ok! I will remove it!